infra/feast-operator/internal/controller/services/services.go

```diff
@@ -286,11 +286,18 @@ func (feast *FeastServices) setDeployment(deploy *appsv1.Deployment, feastType F
 		return err
 	}
 	deploy.Labels = feast.getLabels(feastType)
+	// <<<<<<< HEAD
```

Comment:
need to fix a merge issue here

Reply:
Thanks for catching this. I tested with the conflict commented out, then forgot to remove it.